pipeline-model-definition/src/test/resources/multipleProperties.groovy

```diff
@@ -36,7 +36,7 @@ pipeline {
     options {
         buildDiscarder(logRotator(numToKeepStr:'1'))
         disableConcurrentBuilds()
-        skipDefaultCheckout(true)
+        skipDefaultCheckout()
```

Comment:
Need a couple more tests, specifically ones where you call skip skipDefaultCheckout twice, once with one value then again with a different value.